tez-runtime-library/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1,13 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
does this extra file have any advantages compared to adding this below as a dependency:
```
<dependency>
    <groupId>org.mockito</groupId>
    <artifactId>mockito-inline</artifactId>
    <version>xxx</version>
    <scope>test</scope>
</dependency>
```

Reply:
@abstractdog Both options work, the difference is just the extra file versus extra dependency. As per [this doc](https://javadoc.io/static/org.mockito/mockito-core/4.8.1/org/mockito/Mockito.html#39), `mockito-inline` artifact likely to be discontinued once mocking of final classes and methods gets integrated into the default mock maker. Please let me know if adding `mockito-inline` dependency looks better.

Reply:
thanks for clarifying, I'm fine with this file then